aries-site/src/components/home/Highlights.js

```diff
@@ -0,0 +1,114 @@
+import { highlights } from '../../data';
+
+const HighlightsLayout = () => {
+  const size = useContext(ResponsiveContext);
+  const darkMode = useDarkMode();
+
+  return (
+    <Grid
+      pad={{ horizontal: size !== 'small' ? 'medium' : 'large' }}
+      columns={size !== 'small' ? 'medium' : '100%'}
+      rows={[['auto', 'full']]}
+      gap="large"
+      fill
+      justify="center"
+    >
+      {highlights.map(({ name, summary, image }) => {
+        const href = nameToPath(name);
+        const isInternalLink = internalLink.test(href);
+        const Wrapper = isInternalLink ? Link : Fragment;
+        const wrapperProps = isInternalLink && {
+          href,
+          passHref: true,
+        };
+
+        return (
+          <Wrapper key={name} {...wrapperProps}>
+            <ContentPreviewCard
+              forwardedAs="a"
+              style={{ textDecoration: 'none' }}
+              pad="medium"
+            >
+              <Box direction="row" gap="medium">
+                <Box width="small" round="xsmall">
+                  {image && (
+                    <Image
+                      src={
+                        darkMode.value
+                          ? image.src.dark || image.src
+                          : image.src.light || image.src
+                      }
+                      alt={image.alt}
+                      fit={image.fit || 'contain'}
+                    />
+                  )}
+                </Box>
+                <Box fill>
+                  <Text
+                    weight="bold"
+                    size="large"
+                    margin={{ top: 'small' }}
+                    color="text-strong"
+                  >
+                    {name}
+                  </Text>
+                  <Paragraph>{summary}</Paragraph>
+                </Box>
+              </Box>
+            </ContentPreviewCard>
+          </Wrapper>
+        );
+      })}
+    </Grid>
+  );
+};
+
+export const Highlights = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box
+      fill
+      background="background-front"
+      gap="medium"
+      pad={{ vertical: 'large' }}
+    >
+      <Box
+        justify="center"
+        align="center"
+        pad={{ horizontal: size !== 'small' ? 'xlarge' : 'large' }}
+        gap="large"
+      >
+        <Heading margin="none" level={2} size="large">
+          Highlights
+        </Heading>
+        <Box width="large" pad={{ bottom: 'medium' }}>
+          <Paragraph size="xlarge" fill textAlign="center" margin="none">
+            The HPE Design System team is committed to conducting thorough
+            research so you don't have to think about it. Just find what you
+            need, design and deliver quickly!
+          </Paragraph>
+        </Box>
+      </Box>
+      <HighlightsLayout />
+      <Box fill="horizontal" align="center" justify="center" pad="medium">
+        <Button primary label="Show Me More" />
```

Comment:
Are we going to add functionality to this button on the first pass? If not, I'd recommend we remove it

Reply:
Yup. Done in my sandbox and will be in the next push (goes to the old homepage as we talked about the other day.)